lib/elixir_search_extractor_web/templates/keyword_file/index.html.eex

```diff
@@ -0,0 +1,28 @@
+<h1>Listing Keyword files</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Csv file</th>
+
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for keyword_file <- @keyword_files do %>
```

Comment:
Let's fix indent here

Reply:
@hanam1ni Fixed indent in frontend PR #36